Source/WebKit/Platform/IPC/Connection.cpp

```diff
@@ -1521,8 +1514,9 @@ SerialFunctionDispatcher& Connection::dispatcher()
     // dispatcher can only be accessed while the connection is valid,
     // and must have the incoming message lock held if not being
     // called from the SerialFunctionDispatcher.
-    RELEASE_ASSERT(m_syncState);
-    RefPtr dispatcher = m_syncState->dispatcher();
+    RefPtr syncState = m_syncState;
+    RELEASE_ASSERT(syncState);
+    RefPtr dispatcher = syncState->dispatcher();
```

Comment:
Doesn’t the `RefPtr` `->` operator already do the equivalent of a `RELEASE_ASSERT`?